packages/styles/select.css

```
@@ -85,3 +87,11 @@
 .Field__select--wrapper.Field--has-error select:focus:hover {
   border-color: var(--field-border-color-error-hover);
 }
+
+.cauldron--theme-dark .Field__select--wrapper select:invalid:focus:hover,
+.cauldron--theme-dark
+  .Field__select--wrapper.Field--has-error
+  select:focus:hover {
+  box-shadow: 0px 0px 5px var(--accent-danger),
```

0 values can be unitless.

```suggestion
  box-shadow: 0 0 5px var(--accent-danger),
```